src/olympia/addons/tests/test_models.py

```diff
@@ -1280,18 +1280,18 @@ def test_can_request_review_rejected(self):
         addon.latest_version.files.update(status=amo.STATUS_DISABLED)
         assert addon.can_request_review() == ()
 
-    def check(self, status, exp, kw={}):
+    def check(self, status, exp, no_prelim=False, kw={}):
         addon = Addon.objects.get(pk=3615)
         changes = {'status': status, 'disabled_by_user': False}
         changes.update(**kw)
         addon.update(**changes)
-        assert addon.can_request_review() == exp
+        assert addon.can_request_review(no_prelim) == exp
```

Comment:
Use the keyword argument for clarity ? 

`assert addon.can_request_review(disallow_preliminary_review=no_prelim) == exp` 

Maybe even rename `no_prelim` to `disallow_preliminary_review` as well.